src/common/snippets/tests/src/pass/gn_tokenization.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <pass/gn_tokenization.hpp>
+#include "subgraph_group_normalization.hpp"
+#include "snippets/pass/gn_tokenization.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+
+void TokenizeGNSnippetsTests::run() {
+    ASSERT_TRUE(model);
+    manager.register_pass<ov::snippets::pass::TokenizeGNSnippets>();
+    disable_rt_info_check();
+}
+
+TEST_F(TokenizeGNSnippetsTests, smoke_Snippets_GN_Tokenization_2D) {
+    const auto &f = GroupNormalizationFunction(std::vector<PartialShape>{{1, 10}, {10}, {10}}, 2, 0.00001f);
+    model = f.getOriginal();
+    model_ref = f.getReference();
+    run();
+}
```

Comment:
Maybe we can use parameterized tests here (as in e.g. src/common/snippets/tests/src/pass/softmax_decomposition.cpp)? It's easier and takes less code to extend the parameterized tests. I don't insist though

Reply:
changed to parameterized test cases. Thanks Vladislav!